sdk/servicebus/service-bus/README.md

```diff
@@ -18,10 +18,10 @@ Resources for the v7.0.0-preview.5 of `@azure/service-bus`:
 > **NOTE**: This document has instructions, links and code snippets for the **preview** of the next version of the `@azure/service-bus` package
 > which has different APIs than the stable version. To use the stable version of the library use the below resources.
 
-[Source code or Readme for v1.1.5](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/service-bus_1.1.5/sdk/servicebus/service-bus) |
-[Package for v1.1.5 (npm)](https://www.npmjs.com/package/@azure/service-bus/v/1.1.5) |
-[API Reference Documentation for v1.1.5](https://docs.microsoft.com/javascript/api/%40azure/service-bus/?view=azure-node-latest) |
-[Samples for v1.1.5](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/service-bus_1.1.5/sdk/servicebus/service-bus/samples)
+[Source code or Readme for v1.1.9](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/service-bus_1.1.9/sdk/servicebus/service-bus) |
+[Package for v1.1.9 (npm)](https://www.npmjs.com/package/@azure/service-bus/v/1.1.9) |
+[API Reference Documentation for v1.1.9](https://docs.microsoft.com/javascript/api/%40azure/service-bus/?view=azure-node-latest) |
+[Samples for v1.1.9](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/service-bus_1.1.9/sdk/servicebus/service-bus/samples)
```

Comment:
Let's change the samples to point to master/samples-v1 instead and just not have to worry about the hotfix branch.

Reply:
This isn't the hotfix branch, we are using the release tag.
It should be fine to keep 1.1.9 here since we are also pointing the readme, npm link and ref docs

Reply:
I know it'll work. I was thinking it'd be nice if we stop pointing to the old reference to make our lives easier (so we can update the track 1 samples independent of releasing the package).

Reply:
Fine, updated :)

Reply:
FINE!